spec/models/company_spec.rb

```diff
@@ -1,5 +1,8 @@
 require "rails_helper"
 
 RSpec.describe Company, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "validations" do
+    it { should have_many(:jobs) }
+    it { should validate_presence_of(:name) }
```

Comment:
Live and in person! Finally got to see them...can't wait to try them out myself now :smile: